hw/top_earlgrey/cdc/cdc_waivers.multclk_crossings.tcl

```diff
@@ -0,0 +1,31 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+set_rule_status -rule {MULTCLK_CROSSINGS} -expression {(ClockDomains == "") && (FromClockSignal == "-") && (MultiClockDomains == "JTAG_TCK::SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK") && (SampleRxSignal == "top_earlgrey.u_spi_device.u_sck_csb_edge.g_sync.u_sync.u_sync_1.gen_generic.u_impl_generic.q_o[0]") && (SampleTxSignal == "u_padring.gen_mio_pads[38].u_mio_pad.gen_generic.u_impl_generic.in_o") && (ToClockSignal == "top_earlgrey.u_spi_device.u_clk_spi_in_buf.clk_o")} -status {Waived} -comment {Reviewed : a normal CDC path handled properly}
```

Comment:
sorry @jeoongp could you elaborate a bit more on how this is handled properly? This will make it a bit easier to digest later on and to review.

Reply:
![Screen Shot 2022-12-19 at 3 42 38 PM](https://user-images.githubusercontent.com/103145460/208549179-6492d300-891a-45e4-999e-5a93cbaac11e.png)
PAD inputs (mio_pad[38]) is recognized in JTAG_TCK domain and transferred to SPI clock domain.

Reply:
ah sounds good. Is there any way you could breifly describe this as a block comment?

Reply:
@jeoongp can you provide some more info on this particular path?

I wonder why this pops up, becuase mio_pad[38] (IOR3) is a muxed IO pad, whereas the SPI device clock and associated signals are all dedicated...

Reply:
![Screen Shot 2022-12-21 at 10 33 10 AM](https://user-images.githubusercontent.com/103145460/208978448-7e3152a1-4724-49d3-b4e9-61f2cef5cdea.png)

Reply:
set JTAG_CLK_PIN IOR3 as defined in SDC

Reply:
> ah sounds good. Is there any way you could breifly describe this as a block comment?

@tjaychen I am not clear of block comment.. Do you mean we can add more detailed description about it at the head of the RTL file?

Reply:
ah okay, so this is the tpm csb, which is a muxed connection.
I completely forgot about that. Thanks for the reminder.

Reply:
o no i just meant in the waiver, if you could add a comment explaining these.
Since this seems to apply to a lot of waived messages? I couldn't tell exactly what this was based on the existing comment.

Reply:
Yes. The existing comment of this waiver is W_DATA RuleDataId 115. I can manually find and add it on each waiver.

Reply:
o no it doesn't have to be on each waiver. That's what i meant by "block".
If a bunch of waivers are all related to the same thing, you can just add a detailed comment explaining all of them.

I couldn't tell by the message here "handled properly" what it actually meant.  It would be nice to just have some context.

Reply:
> ah okay, so this is the tpm csb, which is a muxed connection. I completely forgot about that. Thanks for the reminder.

Ah got it, now it makes sense, thanks.

Reply:
> o no it doesn't have to be on each waiver. That's what i meant by "block". If a bunch of waivers are all related to the same thing, you can just add a detailed comment explaining all of them.
> 
> I couldn't tell by the message here "handled properly" what it actually meant. It would be nice to just have some context.

Added some block comments